app/views/contexts/summary/list_collector_block.py

```diff
@@ -160,20 +163,27 @@ def _add_link(
             )
 
     def _get_related_answers(
-        self, list_model: ListModel
-    ) -> Optional[dict[str, list[dict[str, Any]]]]:
+        self, list_model: ListModel, repeating_blocks: Sequence[ImmutableDict]
+    ) -> dict[str, list[dict]] | None:
         section_id = self._section["id"]
 
         related_answers = self._schema.get_related_answers_for_list_for_section(
             section_id=section_id, list_name=list_model.name
         )
-        if not related_answers:
+
+        blocks: list[dict] = []
```

Comment:
Wrong type here as it is empty list?

Reply:
its initialised empty but then populated with `_get_blocks_for_related_answers` or `repeating_blocks` so specifying a type is correct, though if repeating blocks are indeed immutable dictionaries then this would be a list of `dict | ImmutableDict`? but I think the type for `repeating_blocks` might be the incorrect one, rather than this

Reply:
Yep will change to list[ImmutableDict], its only initialised empty.

Reply:
done 👍